webcompat/views.py

```diff
@@ -231,10 +231,12 @@ def create_issue():
             ip=request.remote_addr,
             url=form['url'].encode('utf-8')))
         # Checking blacklisted domains
-        if is_blacklisted_domain(urlparse.urlsplit(form['url']).hostname):
-            msg = app.config['IS_BLACKLISTED_DOMAIN'].format(form['url'])
-            flash(msg, 'notimeout')
-            return redirect(url_for('index'))
+        if form.get('submit_type') == PROXY_REPORT:
```

Comment:
@Anushi1998 almost there. In my code suggestion I was proposing also to move the code after the `is_valid_issue` block. :)